src/App.tsx

```diff
@@ -198,9 +203,14 @@ const App: React.FC = () => {
   }
 
   useInterval(() => {
-    election && updateStatus()
-    cardServerAvailable && !election && readCard()
-    document.getElementById('ballotString')!.focus()
+    if (election) {
+      updateStatus()
+      ballotScannerInput &&
+        ballotScannerInput.current &&
+        ballotScannerInput.current.focus()
```

Comment:
Looking forward to optional chaining!

```ts
ballotScannerInput?.current?.focus()
```

Just waiting on React…

Reply:
Ferserious!